mern/client/src/Components/transcript-scroller.component.css

```diff
@@ -0,0 +1,51 @@
+/* width */
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: #e5e5e5;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: rgb(182, 182, 182);
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: rgb(155, 155, 155);
+}
+
+.scroller { 
+  background-color: rgb(221, 221, 221);
+  height: 700px;
+  width: 500px;
+  border-radius: 20px;
+  overflow-y: auto;
+  overflow-x: auto;
+ }
+
+@font-face {
```

Comment:
shoudln't need to import the font in your css file. having it in tailwind.config should be enough

Reply:
Ok thanks!